src/frontend/efiling-frontend/src/domain/package/package-review/PaymentList.js

```diff
@@ -48,6 +50,20 @@ export default function PaymentList({ payments }) {
     setTotal({ stat: subStat.add(csoStat), toDate: subToDate.add(csoToDate) });
   }, [payments]);
 
+  const handleClick = () => {
+    downloadPaymentReceipt(packageId).catch((err) => {
+      errorRedirect(sessionStorage.getItem("errorUrl"), err);
+    });
+  };
+
+  const handleKeyDown = (e) => {
+    if (e && e.keyCode === 13) {
```

Comment:
There is now a util method for this so we can keep the logic in one place.
eventUtil.isEnter(e)